README.md

```diff
@@ -370,6 +370,7 @@ Thanks to all [contributors](https://github.com/thangchung/awesome-dotnet-core/g
 ### Misc
 * [AngleSharp](https://github.com/AngleSharp/AngleSharp) - The ultimate angle brackets parser library. It parses HTML5, MathML, SVG and CSS to construct a DOM based on the official W3C specification. Comparable to beautifulsoup4 of python.
 * [AutoMapper](https://github.com/AutoMapper/AutoMapper) - Convention-based object-object mapper in .NET.
+* [AppCfg.Net](https://github.com/minhhungit/AppCfg.Net) - Type-safe, easy and powerful configuration framework for .NET developers
```

Comment:
we already have built in interface to support this kind of task. I dont think we need this lib for now

Reply:
Hi @ngohungphuc 
> we already have built in interface to support this kind of task

I know, but I don't think it can help me load setting from database. AppCfg.Net can do it easy. It also support `multi-tenancy` for database source.

Besides, AppCfg support **immutable** setting. This help us avoid changing setting value at runtime, for example:
```csharp
public interface ISetting
{
    int Age { get; } // accept
    string Name { get; set; } // accept
}
```

Anyway, AppCfg.Net is just an optional, you can use built-in feature if you think it's good.

Reply:
You can write custom providers for Microsoft.Extensions.Configuration and multi-tenancy is more a question of the DI setup.

I think your package is good, but personally I would not use it because I would have to maintain 2 configuration integrations in my app code.